src/templates/clue_template.py

```diff
@@ -1,4 +1,6 @@
 """
+To get started, check out the "Device Simulator Express: Getting Started" command in the command pallete, which you can access with `CMD + SHIFT + P` For Mac and `CTRL + SHIFT + P` for Windows and Linux.

```

Comment:
We might want to share in the README that you can access the palette from `View -> Comand Palette...` in case the shortcuts have changed for some users. Not something we need to do here or in this change, just something to keep in mind.

Reply:
great suggestion. I added it to the readme